wallet/res/values/strings-extra.xml

```diff
@@ -192,18 +205,18 @@
     <string name="verify_it_is_important_subtitle">We are about to show you the secret key to your wallet.</string>
     <string name="verify_info_text_1">This recovery phrase is your access to the funds in this wallet.</string>
     <string name="verify_info_text_2">We do not store this recovery phrase.</string>
-    <string name="verify_info_text_3">In case if this device is lost / damaged, incase if the dash wallet is uninstalled accidentally from this device, you will need this recovery phrase to access your funds.</string>
+    <string name="verify_info_text_3">You will need this recovery phrase to access your funds if this device is lost, damaged or if Dash Wallet ever is accidentally uninstalled from this device.</string>
     <string name="verify_skip">Skip</string>
-    <string name="verify_secure_now_btn">Secure Now</string>
-    <string name="verify_secure_now_message">If you lose this device, you will lose your funds. Get your UniqueSecretKey so that you can restore your wallet on another device.</string>
-    <string name="verify_seed_title">Please write this down</string>
-    <string name="verify_seed_write_down_confirmation">I have written this down</string>
+    <string name="verify_secure_now_btn">Secure now</string>
+    <string name="verify_secure_now_message">If you lose this device, you will lose your funds. Get your recovery phrase so that you can restore your wallet on another device.</string>
+    <string name="verify_seed_title">Please write it down</string>
+    <string name="verify_seed_write_down_confirmation">I wrote it down</string>
```

Comment:
These match iOS.

The designs use the word "this" instead of "it"

Also the designs used passive voice:  "I have written this down" - These match iOS.